src/components/HeaderSection.vue

```diff
@@ -13,21 +13,39 @@
       <button class="login-btn">Login</button>
       <button class="order-btn">Order Now</button>
     </div>
+    <div class="nav-icon">
```

Comment:
Remove the `@click="toggleIcon"` from lines 17 and 18 and put it on line 16
That way you're not repeating code